packages/moonstone/Panels/Panels.js

```diff
@@ -69,6 +70,15 @@ const PanelsBase = kind({
 		 */
 		closeButtonBackgroundOpacity: PropTypes.oneOf(['opaque', 'translucent', 'lightTranslucent', 'transparent']),
 
+		/**
+		* Sets the hint string read when focusing the application close button.
+		*
+		* @type {String}
+		* @default 'Exit app'
+		* @public
+		*/
+		exitAppAriaLabel: PropTypes.string,
```

Comment:
Suggest using `closeButtonAriaLabel` for consistency with the other related prop names